app/Libraries/UKCP.php

```diff
@@ -140,4 +144,38 @@ public static function getPathForToken($tokenID, $account)
     {
         return self::TOKEN_PATH_ROOT.$account->id.'/'.$tokenID.'.json';
     }
+
+    public function getStandStatus(string $airfield): array
+    {
+        try {
+            $cached = Cache::get($this->getStandStatusCacheKey($airfield));
+            if ($cached) {
+                return $cached;
+            }
+
+            $response = $this->client->get(
+                sprintf('%s/api/stand/status?airfield=%s', config('services.ukcp.url'), $airfield),
+                ['timeout' => 5]
+            );
+            $body = json_decode($response->getBody()->getContents(), true);
+
+            return tap(
+                collect($body['stands'])->sortBy('identifier', SORT_NUMERIC)->values()->toArray(),
+                fn (array $stands) => Cache::put(
+                    $this->getStandStatusCacheKey($airfield),
+                    $stands,
+                    Carbon::parse($body['refresh_at'])
+                )
+            );
+        } catch (ClientException $e) {
+            Log::warning("UKCP Client Error {$e->getMessage()} when getting stand status for {$airfield}");
+
+            return [];
+        }
+    }
+
+    private function getStandStatusCacheKey(string $airfield): string
+    {
+        return sprintf(self::STAND_STATUS_CACHE_FORMAT, $airfield);
```

Comment:
Is there much value extract this onto the class rather than just using a magic string? It feels a bit clunky to read.

Reply:
Yes! 62436dce